app/add_some_abilities/add_potion_abilities.py

```diff
@@ -0,0 +1,3 @@
+def add_ability_from_potion(knight: dict, ability: str) -> None:
```

Comment:
Consider adding type annotations for the `knight` parameter to specify the expected structure of the dictionary. This will improve code readability and help with type checking.